src/AzureExtension/DevBox/DevBoxInstance.cs

```diff
@@ -447,5 +452,9 @@ public IAsyncOperation<ComputeSystemOperationResult> ModifyPropertiesAsync(strin
         }).AsAsyncOperation();
     }
 
-    public IApplyConfigurationOperation ApplyConfiguration(string configuration) => throw new NotImplementedException();
+    public IApplyConfigurationOperation? ApplyConfiguration(string configuration)
+    {
+        // Apply configuration isn't supported yet for Dev Boxes.
+        return null;

```

Comment:
NIT: throw `NotImplementedException` instead.  That way, if a dev is testing DevBox, they are warned if they try using a not implemented feature.